src/loops/simple_programs_with_repitition/hard/q5/NowThatsPerfect.java

```diff
@@ -0,0 +1,38 @@
+package loops.simple_programs_with_repitition.hard.q5;
+
+/*
+Write a program with the class name "NowThatsPerfect" to accept a number from the user and determine whether it is a "Perfect" number.
+(Hint: A perfect number is one whose factors sum up to equal the given number.
```

Comment:
I think we should just define what a perfect number is instead of putting it in a hint (because honestly without the hint I would not know what a perfect number is)